test-ezxmlm/test_ezxmlm_decode.ml

```diff
@@ -225,3 +225,33 @@ let () =
       (* this prints "More data, even more data" *)
   | Error e ->
       failwith @@ string_of_error e
+
```

Comment:
Can you add `@.@.` to the `printf` in the previous test case?

I.e. `printf "%a@.@."`.

(`@.` instructs the formatter to print a newline and flush the out channel.)